### PR TITLE
Dockerize CI.

### LINKED ISSUE #164: Dockerize CI.
Currently we have to fight with Travis Debian based OS to install dependencies for the build, we can move build into docker and do whatever we want there.

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
Closes #164.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,14 @@ We have quite a bunch of samples showing off some practical applications.
 
 [![asciicast demo](samples/demo.png)](https://asciinema.org/a/101327)
 
+### How to build (for contributors)
```
Comment (arturdryomov) [inline README.md]: `How to Run Tests`?

I think this is more correct.

File: README.md [hunk 2]
```diff
@@ -33,6 +33,14 @@ We have quite a bunch of samples showing off some practical applications.
...
+
+Dependencies: you only need `docker` and `bash` installed on your machine.
```
Comment (arturdryomov) [inline README.md]: Dependencies: Bash, Docker.

File: README.md [hunk 3]
```diff
@@ -33,6 +33,14 @@ We have quite a bunch of samples showing off some practical applications.
...
+
+```console
+bash ci/build.sh
```
Comment (arturdryomov) [inline README.md]: `$ bash`?

File: ci/docker/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+ENV ANDROID_SDK_INSTALL_COMPONENT "echo \"y\" | \"$ANDROID_HOME\"/tools/bin/sdkmanager --verbose"
+
+RUN mkdir -p $ANDROID_HOME && \
+	curl https://dl.google.com/android/repository/$ANDROID_SDK_FILE_NAME --progress-bar --location --output $ANDROID_SDK_FILE_NAME && \
+	unzip $ANDROID_SDK_FILE_NAME -d $ANDROID_HOME && \
+	rm $ANDROID_SDK_FILE_NAME && \
+	eval $ANDROID_SDK_INSTALL_COMPONENT '"tools"' && \
+    eval $ANDROID_SDK_INSTALL_COMPONENT '"platform-tools"' && \
+    eval $ANDROID_SDK_INSTALL_COMPONENT '"build-tools;25.0.2"' && \
+    eval $ANDROID_SDK_INSTALL_COMPONENT '"platforms;android-25"'
```
Comment (arturdryomov) [inline ci/docker/Dockerfile]: Indentation is messed up a bit.

**Review by arturdryomov — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (artem-zinnatullin) [reply README.md]: Fair!

File: README.md (hunk 2 above)
Comment (artem-zinnatullin) [reply README.md]: Uh oh ok

**Pull request merged**